reader/utils.go

```diff
@@ -13,14 +15,5 @@ func isProtected(char rune) (ok bool) {
 }
 
 func isWhitespace(char rune) (ok bool) {
```

Comment:
Can you just replace all the usage of `isWhitespace`  with `unicode.IsSpace`?

Reply:
Sure, can do